Dockerfile

```diff
@@ -10,7 +10,9 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1227.1726694542
 ARG conftest_version=0.45.0
 ARG BATS_VERSION=1.6.0
 ARG sbom_utility_version=0.12.0
+# Two OPM versions are required because index images for OCP version >=v4.17 are incompatible with 1.26.3
 ARG OPM_VERSION=v1.26.3
+ARG OPM_VERSION_2=v1.40.0
```

Comment:
Can we not just use the most recent OPM version?

We are also inconsistent with the release utils image: https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L7

Reply:
I agree. If all we're doing is `opm render` to get the raw contents of the FBC, and then jq to read it, we should not  be sensitive to the `opm` version.

Reply:
aha, I tried doing what IIB does to add bundles i.e. 1.26.4 for <4.17 and 1.40.0 for the ones >4.17. I guess it only matters when adding bundles.

I tried rendering 4.14 index with 1.40 and 1.26.4 and the output were same. I'll update the existing OPM version itself to 1.40.0 and use it across all index images. thanks!

Reply:
Done